.scripts/readme.ts

```diff
@@ -102,7 +102,7 @@ export async function getSinglePackageName(typescriptReadmePath: string): Promis
     const readmeBuffer: Buffer = await fs.readFile(typescriptReadmePath);
     const yamlSectionBuffer = await getYamlSection(readmeBuffer, "``` yaml $(typescript)", "```");
     const yamlSectionText = yamlSectionBuffer.toString();
-    const yamlSection = yaml.safeLoad(yamlSectionText);
+    const yamlSection:any = yaml.safeLoad(yamlSectionText);
```

Comment:
why did this have to change?

Reply:
This was causing a build error with the `npm run build` step. The only way that got fixed was to have this. Since this was a harmless code, I let it slide :-)